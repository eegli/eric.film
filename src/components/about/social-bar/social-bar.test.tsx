import { shallow } from '@/src/tests/testUtils';
import SocialBar from './social-bar.component';

describe('About', () => {
  it('renders social bar component', () => {
    const wrapper = shallow(<SocialBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
