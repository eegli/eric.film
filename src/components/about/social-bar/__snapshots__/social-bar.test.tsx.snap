// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders social bar component 1`] = `
<social-barstyles__Container>
  <social-barstyles__IconContainer
    aria-label="Link to Github"
    href="https://github.com/eegli"
    rel="noopener"
    target="_blank"
  >
    <FaGithub />
  </social-barstyles__IconContainer>
  <social-barstyles__IconContainer
    aria-label="Link to Linkedin"
    href="https://www.linkedin.com/in/ericegli/"
    rel="noopener"
    target="_blank"
  >
    <FaLinkedinIn />
  </social-barstyles__IconContainer>
  <social-barstyles__IconContainer
    aria-label="Link to Instagram"
    href="http://instagram.com/eric.film"
    rel="noopener"
    target="_blank"
  >
    <FaInstagram />
  </social-barstyles__IconContainer>
</social-barstyles__Container>
`;
